recipe/meta.yaml

```
@@ -35,19 +35,20 @@ test:
     - jplephem
   commands:
     - curl -sS -O https://naif.jpl.nasa.gov/pub/naif/generic_kernels/spk/planets/a_old_versions/de421.bsp
-    - curl -sS -O https://naif.jpl.nasa.gov/pub/naif/generic_kernels/spk/planets/de430.bsp
     - curl -sS -O https://naif.jpl.nasa.gov/pub/naif/generic_kernels/spk/planets/a_old_versions/de405.bsp
-    - curl -sS -O ftp://ssd.jpl.nasa.gov/pub/eph/planets/test-data/430/testpo.430
-    - python -m unittest discover jplephem.test
-    - python -m jplephem.jpltest
+    - pip install de421
```

The issue is coming from this line. If you click through you can get to the build log which contains:
```
/home/conda/feedstock_root/build_artifacts/jplephem_1578673603855/test_tmp/run_test.sh: line 9: pip: command not found
```
Using `pip` to install dependencies isn't the right thing to do so I'll dig a bit more and make a suggestion of what you need instead.

---

I think you might not need the package at all (and you can probably skip the affected test if not). If you really do need it, it should be added as a conda package.
```suggestion
```

---

If you have problems with source files like 65be17b24457790c5ca0180674ea82aedb1ccadc did, it might be easier to use the source tarball from the upstream repository: `https://github.com/brandon-rhodes/python-jplephem/archive/v{{ version }}.tar.gz`

You can then use [`source_files`](https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#source-files) to copy the test data in `ci` to the testing environment.

---

arg, I was thinking because I was not maintainer I have no access to the logs. 
But I just missed completly the gray text
"View more details on Azure Pipelines"

 may be on error it should have a "red"  background or somehow big sign, look here ;)

 Thx for helping.

---

If there is no other option I may be able to add a package on conda-forge too. I found it on anaconda

https://anaconda.org/search?q=de421

---

For a bunch of technical reasons conda-forge (effectively) requires all packages to be on the `conda-forge` channel.

---

What happens if you omit that package from the recipe? It's only installed during the library's own CI tests to check if it still works against the deprecated system of distributing NASA ephemerides through pip. All modern usage, so far as I know, now encourages folks to download the originals from NASA.

And: there is probably not a high chance that you somehow break jplephem's ability to use those old ephemeris packages by the way you package the library itself.

---

removing it gets the doctests to fail on

ResourceWarning: unclosed file <_io.BufferedReader name='de421.bsp'>
FileNotFoundError: [Errno 2] No such file or directory: 'moon_pa_de421_1900-2050.bpc'

and so on

---

parallel I submit a new recipe.

---

> removing it gets the doctests to fail on

I believe the error you quote to be unrelated to the missing `de421` package. I may be wrong, but I think that error means that you have made progress and are now facing a new problem to solve — because, when I read the doctest (the text of `jplephem/__init__.py`, yes?), I don't see any active doctest that tries using the `de421` package. The line `import de421` is in a literal block that should not get run.

---

This problem can likely be solved by 
```suggestion
    + wget https://naif.jpl.nasa.gov/pub/naif/generic_kernels/pck/moon_pa_de421_1900-2050.bpc
```